src/components/PlaygroundComponent.css

```diff
@@ -203,4 +204,19 @@
   --input-background-color: var(--properties-input-background-color);
   --group-background-color: var(--properties-group-background-color);
   --arrow-hover-background-color: var(--properties-arrow-hover-background-color);
+}
+
+/**
+ * Custom Dragula Styles
+ *
+ * @pinussilvestrus: remove me once this is fixed: https://github.com/bpmn-io/form-js/issues/462
+ */
+.cfp-root .fjs-children.fjs-drag-container {
+  border: 2px dashed transparent;
+}
+
+@media only screen and (min-width: 1650px) {
```

Comment:
Thanks for the

> Oops sorry about that, mouse slip :D
> 
> Looks good to me, definitely better to hook into the events but as a quick fix, this works totally fine.
> 
> To fix your problem with it only working based on the screen size, you could just check the size of 'fjs-container' horizontal margins, and add a class based on that. You'd only need to recalculate the values on [resize](https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event) and on toggling of the preview I think.
> 
> Or if you want to be super rigorous, you could also think about using the [resize observer api](https://developer.mozilla.org/en-US/docs/Web/API/Resize_Observer_API).

Yeah using an observer was exactly I wanted to prevent for this solution. As we have class selectors for this already (cf. https://github.com/camunda/form-playground/pull/41#discussion_r1048290337), it's also not necessary 👍

Reply:
For the cursor, let's do it the same way as we do with the collapsible action cursor (`--collapsible-cursor` variable). This means adding a variable, and setting it to default so that the Web Modeler can override it, but the Desktop Modeler can still avoid using cursors.

Reply:
@christian-konrad I added the styling changes. Feel free to merge and release a patch in case it is necessary while I'm absent (or wait for two weeks after I'm back from vacation).